templates/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
can u change the name of the index.html file in the templates folder so it doesn't conflict with the react index.html file. 

name suggestion: time_generator.html